settings.py

```diff
@@ -24,5 +24,7 @@
 USE_TZ = False
```

Comment:
RECOMMENDATION: It is recommended to set USE_TZ to True. This makes Django use timezone-aware datetime objects. If your application deals with dates and times, it's a good idea to make it timezone aware.